src/deck/drivers/src/lighthouse.c

```diff
@@ -89,7 +89,8 @@ static const DeckDriver lighthouse_deck = {
   .pid = 0x10,
   .name = "bcLighthouse4",
 
-  .usedGpio = 0,  // FIXME: set the used pins
+  .usedGpio = 0,
+  .usedPeriph = DECK_USING_UART1 | DECK_USING_I2C,
```

Comment:
I think LH is currently using only UART. I2C is a possibility in the future.

Reply:
Ok.